api/src/main/java/io/strimzi/api/kafka/model/CertificateAuthority.java

```diff
@@ -60,8 +60,10 @@ public void setGenerateCertificateAuthority(boolean generateCertificateAuthority
         this.generateCertificateAuthority = generateCertificateAuthority;
     }
 
-    @Description("If true then the Certificate Authority certificates secrets owner reference is set to the Kafka object. " +
-            "Otherwise no owner reference is set so deleting the Kafka object won't delete the secrets. " +
+    @Description("If true, then the Cluster and Client CA Secrets are configured with the `ownerReference` set to the `Kafka` resource. " +
```

Comment:
Feel free to ignore, but couple of thoughts:

Are the _thens_ are needed?  Read like if statements. 
Is it clear enough that we're talking about deleting when enabled and disabled? Maybe _If the Kafka resource is deleted when `true`....   /  If the Kafka resource is deleted when `false`....

Reply:
Updated the wording and got rid of the _thens_. Much clearer, thanks.